client/landing/stepper/declarative-flow/internals/steps-repository/sensei-launch/index.tsx

```diff
@@ -37,6 +45,21 @@ const SenseiLaunch: Step = ( { navigation: { submit } } ) => {
 
 	const percentage = useSubSteps(
 		[
+			async function installAndActivateTheme( retries ) {
+				try {
+					await installCourseTheme( siteId );
+					await setThemeOnSite( `${ siteId }`, 'pub/course', selectedVariation );
+					return true;
+				} catch ( responseError: unknown ) {
+					if ( ( responseError as ResponseError )?.error === 'theme_already_installed' ) {
+						await setThemeOnSite( `${ siteId }`, 'pub/course', selectedVariation );
+						return true;
+					}
+				}
+
+				await wait( 3000 );
+				return retries >= 10;
+			},
```

Comment:
Here is where all the magic happens, I've moved all the logic here so that we install and activate the theme at the same time we install the plugins and the rest of things.